Makefile

```diff
@@ -1,30 +1,48 @@
 
 TEST_BUILD_DIR=_test_build
+LITE_BUILD_DIR=_lite_build
 PACKAGES=lacaml lbfgs ocephes
 PACKAGES_TEST=$(PACKAGES) kaputt dsfo
 PACKAGES_COVERED:=$(PACKAGES_TEST) bisect_ppx
-PACKAGES_INSTALL=$(PACKAGES_COVERED)
+PACKAGES_INSTALL=cppo $(PACKAGES_COVERED)
+CPPO_TAG:=-plugin-tag 'package(cppo_ocamlbuild)'
 
 SOURCE_DIRS=/util /unc /stats /cls /rgr /uns
+INSTALL_EXTS=a o cma cmi cmo cmt cmx cmxa cmxs
 
-.PHONY: all clean test build install uninstall setup default doc omltest.native
+.PHONY: all clean test build install uninstall setup default doc omltest.native oml.cmxa oml_lite.cmxa lite
 
 default: build
 
-all: build
-
 # This should be called something else.
 setup:
 	opam pin add dsfo git://github.com/rleonid/dsfo
 	opam install $(PACKAGES_INSTALL)
 
+#### Building
+
 oml.cmxa:
-	ocamlbuild -use-ocamlfind $(foreach package, $(PACKAGES),-package $(package)) -I src/lib oml.cma oml.cmxa oml.cmxs
+	ocamlbuild $(CPPO_TAG) -use-ocamlfind $(foreach package, $(PACKAGES),-package $(package)) -I src/lib oml.cma oml.cmxa oml.cmxs
+
+lite:
+	mv src/lib/_tags src/lib/_tags_orig && \
+	cp src/lib/_lite_tags src/lib/_tags && \
+	ocamlbuild -build-dir $(LITE_BUILD_DIR) $(CPPO_TAG) -tag 'cppo_D(OML_LITE)' -use-ocamlfind -I src/lib oml_lite.cma oml_lite.cmxa oml_lite.cmxs && \
+	mv src/lib/_tags_orig src/lib/_tags || \
+	mv src/lib/_tags_orig src/lib/_tags
 
 build: oml.cmxa
 
+clean:
+	ocamlbuild -clean
+	ocamlbuild -build-dir $(TEST_BUILD_DIR) -clean
+	ocamlbuild -build-dir $(LITE_BUILD_DIR) -clean
```

Comment:
`_driver` directory is no more. I replaced it with `$(TEST_BUILD_DIR)` aka `_test_build`.